From b7e9f53f6c7cc7c607473bc10b33ce0f19c6377b Mon Sep 17 00:00:00 2001
From: satishkumar45 <sksatishkumar133@gmail.com>
Date: Sun, 20 Mar 2022 18:30:03 +0530
Subject: [PATCH] Tool:stranded:exer line added Add appy() function.

Signed-off-by: satishkumar45 <sksatishkumar133@gmail.com>
---
 exer.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/exer.c b/exer.c
index 89a1a93..75da5d8 100644
--- a/exer.c
+++ b/exer.c
@@ -3,5 +3,6 @@ int main()
 {
    printf("welcome\n");
    apply(2);
+   apply(3);
    return 0;
 }
-- 
2.7.4

