From 97983a6f970dba00db9f488f76f7cf2d83530750 Mon Sep 17 00:00:00 2001
From: satishkumar45 <sksatishkumar133@gmail.com>
Date: Sat, 12 Mar 2022 16:18:08 +0530
Subject: [PATCH 1/5] added one line

---
 links/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/links/main.c b/links/main.c
index db2333d..016dfa4 100644
--- a/links/main.c
+++ b/links/main.c
@@ -2,6 +2,7 @@
 int main()
 {
   printf("sampling the things \n ");
+  printf("sampling is more thing\n");
   return 0;
 
 }
-- 
2.7.4

